tests/v2/validation/rbac/globalrolesv2/globalroles_v2_test.go

```diff
@@ -0,0 +1,472 @@
+//go:build (validation || infra.any || cluster.any || extended) && !sanity && !stress
```

Comment:
Just to make sure Priya, this test will run every week?

Reply:
Sorry, I assumed extended referred to long duration tests. This can be run bi-weekly since it takes more than 15 minutes to run. I am going to change the tag to "stress".

Reply:
Ok sounds good, just making sure you understand the differences